scripts/ci/images-v2/build-cuda.sh

```diff
@@ -32,7 +32,10 @@ if ! build_squash \
     ornladios/adios2:ci-spack-el8-cuda-gcc8-base \
     ornladios/adios2:ci-spack-el8-cuda-serial \
     Dockerfile.ci-spack-el8-leaf \
-    "--build-arg=COMPILER_IMG_BASE=${build_conf} --build-arg=COMPILER_SPACK_ID=gcc --build-arg=CUDA_VARIANT=+cuda"
+    "--build-arg=COMPILER_IMG_BASE=${build_conf}
+     --build-arg=COMPILER_SPACK_ID=gcc
+     --build-arg=CUDA_VARIANT=+cuda
+     --build-arg=EXTRA_SPACK_PACKAGES=kokkos@3.6.01%gcc+cuda+wrapper"
```

Comment:
I did not have time to reproduce this to test my theory, but if you are using the wrapper for kokkos, it needs to be used for adios as well. How is adios build in the CI?

Reply:
Here you can change the flags for the CUDA build

https://github.com/ornladios/ADIOS2/blob/master/scripts/ci/cmake-v2/ci-el8-cuda-serial.cmake

Reply:
> You should be able to load kokkos without the wrapper. Did you say you were getting errors?

If I do a `kokkos@3.6.01%gcc+cuda+wrapper` with spack 0.18.1